components/server/src/api/workspace-service-api.ts

```diff
@@ -106,6 +109,44 @@ export class WorkspaceServiceAPI implements ServiceImpl<typeof WorkspaceServiceI
         return response;
     }
 
+    async listWorkspaceSessions(
+        req: ListWorkspaceSessionsRequest,
+        _: HandlerContext,
+    ): Promise<ListWorkspaceSessionsResponse> {
+        const page = parsePagination(req.pagination, 50);
```

Comment:
`pagination.page` is deprecated (ing), we could use `parsePaginationToken` now https://github.com/gitpod-io/gitpod/blob/24dbb321dafbd2329f9a2232fe47cc6d410af92c/components/server/src/api/configuration-service-api.ts#L128C33-L128C53

https://github.com/gitpod-io/gitpod/blob/se/workspace-sessions/components/server/src/api/configuration-service-api.ts#L159-L168

Reply:
given that all the code is using pagination.page (only one exception AFAIK) I think we should reconsider deprecating this.